### DISCUSSION TITLE
Locations of DLLs for VSTs?

### DISCUSSION THREAD

**Discussion by sagetone** [apohl79/audiogridder#872] [Q&A]
Hi, I'm interested in giving Audiogridder a go.  New custom DAW would be the client and older DAW would be the server.  Currently, all VST DLLs reside on the client machine.  Does Audiogridder on the server copy DLLs from the client and run them locally on the server machine?  Or would I need to duplicate DLLs over on the server machine?  (The later certainly doable but a bit of a pain because of licensing, registrations, things like iLok, etc)

Thank you for helping this audio/MIDi networking noob.

**Comment by emack999** [comment]
The vst's would reside on the server machine, so you have to install them there.

**Reply by sagetone** [reply]
Thanks emack999.  I was hoping not, but I get it.  I'll just need to get selective on which VST instruments and FX to run on the server then.

**Reply by emack999** [reply]
Yes, pick one or two that use a lot of CPU cycles.   That way you can get a feel for how it all works, while seeing how much less stressed your client machine is.

**Reply by sagetone** [reply]
Yep, will not be bringing all of them over for sure.

**Comment by oldtos** [comment]
The DAW you are using gets the plugin......The server goes on the spare/old computer.

**Reply by sagetone** [reply]
> The DAW you are using gets the plugin......The server goes on the spare/old computer.

Thanks oldtos. So are you saying the plugs/DLLs _**don't**_ need to reside on the server? That's different from what emack999 is advising above.

**Reply by emack999** [reply]
You put the plugins on the server machine. Otherwise, what good would another machine do? :)  The whole point is that the server machine hosts the plugins and bears the CPU weight, while the client machine has more CPU resources to use in its other DAW-related work.
Read here:  https://audiogridder.com/server/ and https://audiogridder.com/plugin/

**Reply by oldtos** [reply]
I should have made it clearer... by plugin I mean Audiogridder plugin... goes on the DAW computer.. The sever goes on the "slave" computer..This is the computer where you want to share VST ect from..  so in you case
"New custom DAW would be the client....(it gets the audiogridder plugin" ...."and older DAW would be the server. (it gets the audiogridder server" ...

**Reply by sagetone** [reply]
Thanks oldtos and emack999, I'm clear now!
